src/extension/devtools/panel.ts

```diff
@@ -15,7 +15,7 @@ panelWindow.on("initializePanel", (message) => {
 panelWindow.on("devtoolsStateChanged", (message) => {
   devtoolsState(message.state);
 
-  if (message.state === "connected") {
+  if (message.state !== "connected") {
```

Comment:
This was added in https://github.com/apollographql/apollo-client-devtools/pull/1379/files#diff-7c1d36e9e85ad36384132856e68b4e2eaf0d11d4d64e911381c8878329a5f38aR18, but removed when we reverted that release. I'm adding it back here since we want this behavior. See my [original comment](https://github.com/apollographql/apollo-client-devtools/pull/1379/files#r1607185319) for more information on the context of this change.